sdk/appcenter-auth/src/test/java/com/microsoft/appcenter/auth/AuthTest.java

```diff
@@ -149,6 +153,34 @@ private static void mockHttpCallSuccess(JSONObject jsonConfig, ServiceCallback s
         serviceCallback.onCallSucceeded(jsonConfig.toString(), headers);
     }
 
+    @Test
+    public void forwardMsalLogging() {
+        AppCenterLog.setLogLevel(VERBOSE);
+        Auth.getInstance();
+        String tag = "msalTag", expectedTag = LOG_TAG + TAG_DELIMITER + tag, message = "Message from MSAL";
+        com.microsoft.identity.common.internal.logging.Logger.verbose(tag, message);
+        verifyStatic();
+        AppCenterLog.verbose(eq(expectedTag), contains(message));
```

Comment:
Can we use eq for message? We don't modify the message so I guess its the exact same and no need to use contains, unless I missed something?

Reply:
MSAL decorates the message with some context.

Reply:
That would for instance become `[2019-08-08 23:09:08 - {"thread_id":"669","correlation_id":"bbb281b5-3142-4440-9fb1-da292b4bbad3"}] Message from MSAL`
`

Reply:
ok